src/main/java/com/synopsys/integration/alert/workflow/startup/AlertStartupInitializer.java

```diff
@@ -153,12 +154,14 @@ private void updateConfigurationFields(final String descriptorName, final boolea
         throws AlertDatabaseConstraintException {
         if (!configurationModels.isEmpty()) {
             if (!foundConfigurationModels.isEmpty()) {
```

Comment:
We no longer need this line. Instead of the `.orElseThrow()` on line 161, we should get the optional and use that for the if-statement.